### PR TITLE
Move STDIO initialization to libcontainer.Process

### PR DESCRIPTION
**Pull request by crosbymichael**
Signed-off-by: Michael Crosby <crosbymichael@gmail.com>

### CODE REVIEW

File: libcontainer/process.go [hunk 1]
```diff
@@ -78,6 +79,54 @@ func (p Process) Signal(sig os.Signal) error {
...
+// IO holds the process's STDIO
+type IO struct {
+	Stdin  io.WriteCloser
+	Stdout io.ReadCloser
+	Stderr io.ReadCloser
+}
+
+// InitalizeIO creates pipes for use with the process's STDIO
+// and returns the opposite side for each
+func (p *Process) InitalizeIO(rootuid int) (i *IO, err error) {
```
Comment (LK4D4) [inline libcontainer/process.go]: Misspell `InitAlize`

**Comment by hqhq** [comment]
@crosbymichael Can you describe in the changelog what problem this PR fixes or it's just refactor?

**Comment by crosbymichael** [comment]
@hqhq it's just a refactor

**Comment by LK4D4** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**